movie.go

```diff
@@ -78,6 +86,18 @@ func getMovieWithDetail(id int) MovieDetails {
 				} else {
 					var actors Actors
 					json.Unmarshal(body, &actors)
+
+					var crew Crew
+					json.Unmarshal(body, &crew)
+
+					for i := 0; i < len(crew.EntireCrew); i++ {
+						fmt.Println(crew.EntireCrew[i].Name + " " + crew.EntireCrew[i].Job)
+						if crew.EntireCrew[i].Job == "Producer" || crew.EntireCrew[i].Job == "Executive Producer" {
```

Comment:
Am I correct in assuming this is a massive list?

Reply:
It depends on the movie, for some movies the list is barely 5 people, but for some of the bigger movies the list can go so far as about 100-200 people (don't quote me on exact numbers)

Reply:
Nice thing is producers usually appear at the top so even massive lists tend to only have a few iterations